### PR TITLE
Gradle 5.6.3

### PR DESCRIPTION
**Pull request by keeganwitt**

### CODE REVIEW

File: library/gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Maintainers: Keegan Witt <keeganwitt@gmail.com> (@keeganwitt)
 GitRepo: https://github.com/keeganwitt/docker-gradle.git
-GitCommit: fee194c2fda18bcd3d2c633b2d1639c0857515eb
+GitCommit: b277db699944f9fc912baa2e70ba5a13fe4963d6
 
 Tags: 5.6.2-jdk8, 5.6-jdk8, jdk8, 5.6.2-jdk, 5.6-jdk, jdk, 5.6.2, 5.6, latest
```
Comment (tianon) [inline library/gradle]: Version numbers in the tag names should be bumped too, right? :sweat_smile:

Comment (keeganwitt) [reply library/gradle]: Good catch, yes 🤦‍♂

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 99f2e74..8c53040 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -7,15 +7,15 @@ gradle:5.6-jre
 gradle:5.6-jre8
 gradle:5.6-jre11
 gradle:5.6-jre12
-gradle:5.6.2
-gradle:5.6.2-jdk
-gradle:5.6.2-jdk8
-gradle:5.6.2-jdk11
-gradle:5.6.2-jdk12
-gradle:5.6.2-jre
-gradle:5.6.2-jre8
-gradle:5.6.2-jre11
-gradle:5.6.2-jre12
+gradle:5.6.3
+gradle:5.6.3-jdk
+gradle:5.6.3-jdk8
+gradle:5.6.3-jdk11
+gradle:5.6.3-jdk12
+gradle:5.6.3-jre
+gradle:5.6.3-jre8
+gradle:5.6.3-jre11
+gradle:5.6.3-jre12
 gradle:jdk
 gradle:jdk8
 gradle:jdk11
diff --git a/gradle_jdk11/Dockerfile b/gradle_jdk11/Dockerfile
index 313f6f5..31fd2b7 100644
--- a/gradle_jdk11/Dockerfile
+++ b/gradle_jdk11/Dockerfile
@@ -32,8 +32,8 @@ RUN apt-get update \
         subversion \
     && rm -rf /var/lib/apt/lists/*
 
-ENV GRADLE_VERSION 5.6.2
-ARG GRADLE_DOWNLOAD_SHA256=32fce6628848f799b0ad3205ae8db67d0d828c10ffe62b748a7c0d9f4a5d9ee0
+ENV GRADLE_VERSION 5.6.3
+ARG GRADLE_DOWNLOAD_SHA256=60a6d8f687e3e7a4bc901cc6bc3db190efae0f02f0cc697e323e0f9336f224a3
 RUN set -o errexit -o nounset \
     && echo "Downloading Gradle" \
     && wget --no-verbose --output-document=gradle.zip "https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-bin.zip" \
diff --git a/gradle_jdk12/Dockerfile b/gradle_jdk12/Dockerfile
index 80bb912..a2eb729 100644
--- a/gradle_jdk12/Dockerfile
+++ b/gradle_jdk12/Dockerfile
@@ -32,8 +32,8 @@ RUN apt-get update \
         subversion \
     && rm -rf /var/lib/apt/lists/*
 
-ENV GRADLE_VERSION 5.6.2
-ARG GRADLE_DOWNLOAD_SHA256=32fce6628848f799b0ad3205ae8db67d0d828c10ffe62b748a7c0d9f4a5d9ee0
+ENV GRADLE_VERSION 5.6.3
+ARG GRADLE_DOWNLOAD_SHA256=60a6d8f687e3e7a4bc901cc6bc3db190efae0f02f0cc697e323e0f9336f224a3
 RUN set -o errexit -o nounset \
     && echo "Downloading Gradle" \
     && wget --no-verbose --output-document=gradle.zip "https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-bin.zip" \
diff --git a/gradle_jre/Dockerfile b/gradle_jre/Dockerfile
index c91d456..99cff8a 100644
--- a/gradle_jre/Dockerfile
+++ b/gradle_jre/Dockerfile
@@ -32,8 +32,8 @@ RUN apt-get update \
         subversion \
     && rm -rf /var/lib/apt/lists/*
 
-ENV GRADLE_VERSION 5.6.2
-ARG GRADLE_DOWNLOAD_SHA256=32fce6628848f799b0ad3205ae8db67d0d828c10ffe62b748a7c0d9f4a5d9ee0
+ENV GRADLE_VERSION 5.6.3
+ARG GRADLE_DOWNLOAD_SHA256=60a6d8f687e3e7a4bc901cc6bc3db190efae0f02f0cc697e323e0f9336f224a3
 RUN set -o errexit -o nounset \
     && echo "Downloading Gradle" \
     && wget --no-verbose --output-document=gradle.zip "https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-bin.zip" \
diff --git a/gradle_jre11/Dockerfile b/gradle_jre11/Dockerfile
index 0c970e0..996b17b 100644
--- a/gradle_jre11/Dockerfile
+++ b/gradle_jre11/Dockerfile
@@ -32,8 +32,8 @@ RUN apt-get update \
         subversion \
     && rm -rf /var/lib/apt/lists/*
 
-ENV GRADLE_VERSION 5.6.2
-ARG GRADLE_DOWNLOAD_SHA256=32fce6628848f799b0ad3205ae8db67d0d828c10ffe62b748a7c0d9f4a5d9ee0
+ENV GRADLE_VERSION 5.6.3
+ARG GRADLE_DOWNLOAD_SHA256=60a6d8f687e3e7a4bc901cc6bc3db190efae0f02f0cc697e323e0f9336f224a3
 RUN set -o errexit -o nounset \
     && echo "Downloading Gradle" \
     && wget --no-verbose --output-document=gradle.zip "https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-bin.zip" \
diff --git a/gradle_jre12/Dockerfile b/gradle_jre12/Dockerfile
index 6ee050c..3095afc 100644
--- a/gradle_jre12/Dockerfile
+++ b/gradle_jre12/Dockerfile
@@ -32,8 +32,8 @@ RUN apt-get update \
         subversion \
     && rm -rf /var/lib/apt/lists/*
 
-ENV GRADLE_VERSION 5.6.2
-ARG GRADLE_DOWNLOAD_SHA256=32fce6628848f799b0ad3205ae8db67d0d828c10ffe62b748a7c0d9f4a5d9ee0
+ENV GRADLE_VERSION 5.6.3
+ARG GRADLE_DOWNLOAD_SHA256=60a6d8f687e3e7a4bc901cc6bc3db190efae0f02f0cc697e323e0f9336f224a3
 RUN set -o errexit -o nounset \
     && echo "Downloading Gradle" \
     && wget --no-verbose --output-document=gradle.zip "https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-bin.zip" \
diff --git a/gradle_latest/Dockerfile b/gradle_latest/Dockerfile
index 0c1a769..d15e82a 100644
--- a/gradle_latest/Dockerfile
+++ b/gradle_latest/Dockerfile
@@ -32,8 +32,8 @@ RUN apt-get update \
         subversion \
     && rm -rf /var/lib/apt/lists/*
 
-ENV GRADLE_VERSION 5.6.2
-ARG GRADLE_DOWNLOAD_SHA256=32fce6628848f799b0ad3205ae8db67d0d828c10ffe62b748a7c0d9f4a5d9ee0
+ENV GRADLE_VERSION 5.6.3
+ARG GRADLE_DOWNLOAD_SHA256=60a6d8f687e3e7a4bc901cc6bc3db190efae0f02f0cc697e323e0f9336f224a3
 RUN set -o errexit -o nounset \
     && echo "Downloading Gradle" \
     && wget --no-verbose --output-document=gradle.zip "https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-bin.zip" \
````

</details>

**Comment by tianon** [comment]
LGTM

Build test of #6806; 2fbdb9cf; `amd64` (`gradle`):

```console

$ bashbrew build gradle:5.6.3-jdk8
Using bashbrew/cache:7b9e63a16f3a45bc443a106e8d5ff618edd27f10aaec6ae180d3e8446f98331e (gradle:5.6.3-jdk8)
Tagging gradle:5.6.3-jdk8
Tagging gradle:5.6-jdk8
Tagging gradle:jdk8
Tagging gradle:5.6.3-jdk
Tagging gradle:5.6-jdk
Tagging gradle:jdk
Tagging gradle:5.6.3
Tagging gradle:5.6
Tagging gradle:latest

$ test/run.sh gradle:5.6.3-jdk8
testing gradle:5.6.3-jdk8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gradle:5.6.3-jre8
Using bashbrew/cache:66968cb5034fbc454cf301e1d9c10ed11df5acb3ec5f0acde68a66640a1e1f3f (gradle:5.6.3-jre8)
Tagging gradle:5.6.3-jre8
Tagging gradle:5.6-jre8
Tagging gradle:jre8
Tagging gradle:5.6.3-jre
Tagging gradle:5.6-jre
Tagging gradle:jre

$ test/run.sh gradle:5.6.3-jre8
testing gradle:5.6.3-jre8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gradle:5.6.3-jdk11
Using bashbrew/cache:cbe8b6fd68e5ee751452813f6f28d43ea0395c6313f6b373dfb0fb9e0936400e (gradle:5.6.3-jdk11)
Tagging gradle:5.6.3-jdk11
Tagging gradle:5.6-jdk11
Tagging gradle:jdk11

$ test/run.sh gradle:5.6.3-jdk11
testing gradle:5.6.3-jdk11
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gradle:5.6.3-jre11
Using bashbrew/cache:e1890096a2c323cdb52bc9f022e709da674c1a0d63b2e7055c47c2484a8bab63 (gradle:5.6.3-jre11)
Tagging gradle:5.6.3-jre11
Tagging gradle:5.6-jre11
Tagging gradle:jre11

$ test/run.sh gradle:5.6.3-jre11
testing gradle:5.6.3-jre11
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gradle:5.6.3-jdk12
Using bashbrew/cache:1692db7bcd3463f1b3c5c3b7076dc6cf6d853dfd075c86405986855fbdfe3cd1 (gradle:5.6.3-jdk12)
Tagging gradle:5.6.3-jdk12
Tagging gradle:5.6-jdk12
Tagging gradle:jdk12

$ test/run.sh gradle:5.6.3-jdk12
testing gradle:5.6.3-jdk12
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gradle:5.6.3-jre12
Using bashbrew/cache:0a8ca6c4d4a3e16bf14c8aae4780a3fc2e3c1e1f657c4c8c63585fa04492a6a4 (gradle:5.6.3-jre12)
Tagging gradle:5.6.3-jre12
Tagging gradle:5.6-jre12
Tagging gradle:jre12

$ test/run.sh gradle:5.6.3-jre12
testing gradle:5.6.3-jre12
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**